core/prelude-linux.el

```diff
@@ -37,7 +37,8 @@
 (prelude-require-packages '(exec-path-from-shell))
 
 (require 'exec-path-from-shell)
-(exec-path-from-shell-initialize)
+(when (display-graphic-p)
```

Comment:
I guess you should put the require inside the `when` as well. I'd also add some comment here. Frankly, I don't even remember why we need this on Linux. I know it's an issue for macOS users, but I don't remember what was the deal with Linux users.